packages/react/src/components/field/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './field';
```

Comment:
So far we dont have any indexes on component level.

I would suggest to either remove this and follow what was done on the IndicatorContainer internal component (so `useField` is exported via `field.tsx`. This would be my preference, because most likely when using Field you will also use `useField`.

If you disagree, then maybe worth removing a component export here, so index would only export any additions. And then add index onto `internal/indicator-container` also.

Reply:
Fair enough, I don't mind removing the index, your first suggestion.